ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/behaviorTree/BehaviorTreeRootNodeState.java

```diff
@@ -22,8 +20,8 @@ public class BehaviorTreeRootNodeState extends BehaviorTreeNodeState<BehaviorTre
    private final BehaviorTreeRootNodeDefinition definition;
    private final CRDTBidirectionalBoolean automaticExecution;
    private final CRDTBidirectionalInteger executionNextIndex;
-   private final CRDTUnidirectionalNotification manualExecutionRequested;
-   private final CRDTUnidirectionalBoolean concurrencyEnabled;
+   private final CRDTBidirectionalNotification manualExecutionRequested;
+   private final CRDTBidirectionalBoolean concurrencyEnabled;
```

Comment:
All fields are now bidirectional to allow the robot to modify it's own tree. Important for testing and AI research.